backend/src/main/java/com/webappsbusters/packagemanagement/controller/ParcelController.java

```diff
@@ -0,0 +1,28 @@
+package com.webappsbusters.packagemanagement.controller;
+
+import com.webappsbusters.packagemanagement.domain.ParcelDto;
+import com.webappsbusters.packagemanagement.exceptions.ParcelNotFoundException;
+import com.webappsbusters.packagemanagement.mapper.ParcelMapper;
+import com.webappsbusters.packagemanagement.service.ParcelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1")
+public class ParcelController {
+
+    private final ParcelService parcelService;
+    private final ParcelMapper parcelMapper;
+
+    @Autowired
+    public ParcelController(ParcelService parcelService, ParcelMapper parcelMapper) {
+        this.parcelService = parcelService;
+        this.parcelMapper = parcelMapper;
+    }
+
+    @GetMapping(value = "parcels/{parcelId}")
+    public ParcelDto getParcel(@PathVariable String parcelId) throws ParcelNotFoundException {
```

Comment:
For good practice use `RestEntity<ParcelDto>`

Reply:
I think you meant ResponseEntity. Done

Reply:
yes